Update webostv with more real-life example of turn on automation

igrybkov opened the pull request:
## Proposed change

The current example of the turn-on automation is broken and not working. I'm providing the one that actually works and could be easily copy-pasted with a change of identifiers


## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

igrybkov committed:
Update webostv with more real-life example of turn on automation

thecode commented:
I think it is good to add the `alias` to the example, but in my opinion `device_id` is mostly for automations from the UI and not using `YAML`, when using `YAML` most users will probably use `entity_id` as the trigger.

thecode commented on source/_integrations/webostv.markdown:
```diff
@@ -49,14 +49,17 @@ Automations can also be created using an automation action:
-        mac: AA-BB-CC-DD-EE-FF
+  - alias: Turn On Living Room TV with WakeOnLan
+    trigger:
+      - platform: device
```
Added @frenck for an additional opinion since I have less experience in this

frenck replied:
I'm not understanding why we should use device automation here?
in general, device automation are less friendly in YAML use case.

Could you explain what was wrong with the existing automation @igrybkov?

igrybkov committed:
Merge branch 'current' into patch-1

igrybkov committed:
Roll back to webostv.turn_on trigger with entity_id

igrybkov commented:
@frenck, well actually nothing is wrong, that's my mistake here. I think I've seen Home Assistant complaining about incorrect platform, but it's probably was something wrong with my environment or I just trusted vscode plugin validation without checking — either of these. I've just double checked the automation as it is and it works well with `webostv.turn_on` as a platform. Sorry about that!

I've rolled back my changes regarding the trigger, but since PR and we all already here, per @thecode's suggestion, I've updated PR to add alias to the automation and also updated Mac address example to match the actual Mac address format because that's how it's used in [`wake_on_lan` documentation](https://www.home-assistant.io/integrations/wake_on_lan/).
These are cosmetic changes now without actual contribution, so feel free to close the PR if you want.

thecode reviewed (dismissed):
Thanks @igrybkov 👍

frenck commented on source/_integrations/webostv.markdown:
```diff
@@ -49,13 +49,14 @@ Automations can also be created using an automation action:
-    - service: wake_on_lan.send_magic_packet
-      data:
-        mac: AA-BB-CC-DD-EE-FF
+  - alias: Turn On Living Room TV with WakeOnLan
```
```suggestion
  - alias: "Turn On Living Room TV with WakeOnLan"
```

frenck committed:
Styleguide tweak

frenck commented:
Thanks @igrybkov 👍
